### DISCUSSION TITLE
terminal lost the opacity adjusting bar

### DISCUSSION THREAD

**Discussion by Dorfeuheinz** [linuxdeepin/developer-center#3629] [Q&A | 问答板块]
when i upgraded my system to 20.8, my terminal & text editor were no longer part of the os. I had to externally download them & when I did, the opacity adjusting feature of the terminal was gone. How do I fix this? i tried changing the terminal config file as mentioned in the git readme.md, but didnt seem to do anything. While the terminal is stationary it has 100% opacity. How do I adjust it???

**Comment by BLumia** [comment]
I saw you created https://github.com/linuxdeepin/developer-center/issues/3631. Since you mention you externally downloaded them, does it mean you compiled it from source? Or does it means you use `sudo apt install deepin-terminal` to manually install them?

**Comment by BLumia** [comment]
According to https://github.com/linuxdeepin/developer-center/issues/3631, using the version from deepin package repo will be fine. Simply `sudo apt install deepin-terminal` will do the trick.
